### PR TITLE
Validator UI contribution guidelines

### PR DESCRIPTION
**Pull request by jenriquesoriano**
Contribution guidelines provided to manage Validator UI source code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,45 @@
...
-
-This repository includes the code of the [User Interface of the INSPIRE Reference Validator](https://inspire.ec.europa.eu/validator).
+# User Interface of the INSPIRE Reference Validator  
+  
+This repository includes the code of the [User Interface of the INSPIRE Reference Validator](https://inspire.ec.europa.eu/validator).  
+  
+# How to contribute to the User Interface of the INSPIRE Reference Validator  
+  
+The process to contribute to the product and integrate developments into the INSPIRE Reference Validator UI is defined by development guidelines which include issue creation, branch management, code review, and deployment strategies.   
+These guidelines are designed to guide users who will contribute to the maintenance and evolution of the the INSPIRE Reference Validator UI, promoting collaborative development practices and facilitating community engagement.
```
Comment (MarcoMinghini) [inline README.md]: fix the repetition "the the"

File: README.md [hunk 2]
```diff
@@ -1,3 +1,45 @@
...
+  
+4. Pull Request (PR) process  
+  
+Once development work is concluded and locally verified, a pull request must be opened targeting the staging branch from the "issue-xxx" branch.    
+A reviewer will be assigned to the PR to ensure code quality and compliance.  
+  
+5. Review and deployment  
+  
+The assigned reviewer will check the code for compliance before approving the PR and then merging the branch to the staging branch.  
+Upon approval, the changes will be deployed to the Staging environment.
```
Comment (MarcoMinghini) [inline README.md]: add link for "Staging environment"

File: README.md [hunk 3]
```diff
@@ -1,3 +1,45 @@
...
+  
+6. Community verification  
+  
+The community will be informed about the changes through the issue created initially by the update of the respective issue.  
+The community will verify the functionality and compliance of the developments in the Staging environment by using it and, in case, reporting additional feedback.  
+  
+7. Production release  
+  
+The staging branch will serve as the source for production releases.  
+The latest version of the staging branch will be deployed in the Staging environment and, when validated the proper functioning, in the Production environment according to the INSPIRE Reference Validator release strategy.
```
Comment (MarcoMinghini) [inline README.md]: add links for "Staging environment" and "release strategy".

**Comment by MarcoMinghini** [comment]
The text added is ok, I only made some minor comments. Feel free to merge the PR once these are addressed.

File: README.md (hunk 1 above)
Comment (jenriquesoriano) [reply README.md]: fixed the repetition "the the"

File: README.md (hunk 3 above)
Comment (jenriquesoriano) [reply README.md]: added links for "Staging environment" and "release strategy".

File: README.md (hunk 2 above)
Comment (jenriquesoriano) [reply README.md]: added links for "Staging environment"

**Pull request merged**